addons/base_import/static/src/additionalFileManager.js

```diff
@@ -0,0 +1,93 @@
+import { checkFileSize } from "@web/core/utils/files";
+
+export class AdditionalFileManager {
+    constructor(resModel, fields, context, options, orm, notificationService, maxBatchSize) {
```

Comment:
I would put maxBatchSize in options

Reply:
and maybe rename options into params as those "options" don't seem to be optional